api/pbaAccounts/index.ts

```diff
@@ -0,0 +1,52 @@
+import { AxiosPromise } from 'axios'
+import * as express from 'express'
+import { environmentConfig } from '../lib/environment.config'
+import { http } from '../lib/http'
+
+async function handleAddressRoute(req, res) {
+  let errReport: any
+  if (!req.query.accountNames) {
+    errReport = {
+      apiError: 'Account is missing',
+      apiStatusCode: '400',
+      message: 'Fee And Pay route error',
+    }
+    res.status(500).send(errReport)
+  }
+  const accountNames = req.query.accountNames.split(',')
+  const accounts = []
+  const accountPromises = new Array<AxiosPromise<any>>()
+  accountNames.forEach((accountName: string) => accountPromises.push(getAccount(accountName)))
+
+  try {
+    console.log(accounts)
+    await Promise.all(accountPromises).catch(err => err).then(allAccounts => {
+      allAccounts.forEach(account => {
+        const data = account.data === 'Account not found' ? {account_name: 'account not found'} : account.data
```

Comment:
I remember we get a 404 error if account name could not be found. Not sure if account.data does return if Account not found.

Reply:
yes it does find account not found and returns 200 for some odd reason so i am sending it directly to FE

Reply:
no worries

Reply:
If you have already tested an Account name and it works that ok. Thanks